src/Session.cpp

```diff
@@ -92,14 +93,32 @@ Session::Session(const Session &s)
     clean();
 }
 
+void Session::deepCopyUsers(const std::unordered_map<std::string, User*>& newUsers)
+{
+    for(auto u: newUsers)
+    {
+        *(userMap[u.first()]) = *(u.second());
```

Comment:
can't find in the reference what is u.first().
I saw (*u).first
can you explain?

Reply:
u is a pair<>, not a pointer. So the usage is fine